apps/financial-aid/web-veita/src/routes/Supervisors/supervisors.tsx

```diff
@@ -9,13 +9,7 @@ import {
   TableHeaders,
   TextTableItem,
 } from '@island.is/financial-aid-web/veita/src/components'
-import {
-  Text,
-  Box,
-  Button,
-  toast,
-  ToastContainer,
-} from '@island.is/island-ui/core'
+import { Text, Box, Button, ToastContainer } from '@island.is/island-ui/core'
```

Comment:
remove ToastContainer, it is not used ? 👁️

Reply:
It is used on line 159